addon/modifiers/enable-tooltip.js

```diff
@@ -0,0 +1,38 @@
+import { setModifierManager } from '@ember/modifier';
+import $ from 'jquery';
+
+export default setModifierManager(
+  () => ({
+    createModifier() {
+      return {
+        element: null
+      };
+    },
+
+    installModifier(state, element, args) {
+      state.element = element;
+      args = args.named;
+
+      $(element).tooltip({
+        delay: {show: 300},
+        placement: args.placement || 'bottom',
+        title: args.title || '',
+        html: args.html || false,
+        trigger: args.trigger || 'hover focus'
+      });
+    },
+
+    updateModifier(state, args) {
+      //We use this function when tooltip title is updated and depends to translation with value
+      args = args.named;
```

Comment:
`const { title } = args.named` then use `title` instead of overwriting a passed argument